scss/_forms.scss

```diff
@@ -731,9 +753,13 @@ p.explain {
     line-height: 1.5;
   }
 
-  strong { color: #000; }
+  strong {
+    color: #000;
```

Comment:
Color literals like `#000` should only be used in variable declarations; they should be referred to via variable everywhere else.